src/tests/specs/workItemTableTest.spec.ts

```diff
@@ -112,4 +112,11 @@ describe('Work Item datatable list: ', () => {
     await browser.sleep(3000);
     expect(await planner.workItemList.hasWorkItem(c.newWorkItem1.title)).toBeTruthy();
   })
+
+  it('clicking on label should filter the workitem list by label', async() => {
+    await planner.workItemList.clickWorkItemLabel(c.workItemTitle7);
+    expect(await planner.header.getFilterConditions()).toContain('label: '+c.label2);
+    await planner.header.clickShowTree();
+    expect(await planner.header.getFilterConditions()).toContain('label: '+c.label2);
```

Comment:
This is not how you would compare the value. It should be
`expect(await planner.header.getFilterConditions()).toContain(c.label2);`